vllm/transformers_utils/config.py

```diff
@@ -662,6 +662,53 @@ def load_params_config(model: Union[str, Path], revision: Optional[str],
             f"and if the config file exists.")
     assert isinstance(config_dict, dict)
 
+    # Load standard Hugging Face config.json to get better defaults
+    # if params.json (loaded into config_dict) is missing these specific keys.
+    hf_standard_config_data = get_hf_file_to_dict("config.json", model,
+                                                  revision)
```

Comment:
This currently always loads and inspects the config.json. Instead, we should only do this extra lookup if `config_dict` is missing `max_position_embeddings`.